app/controllers/posts_controller.rb

```diff
@@ -7,15 +7,15 @@ def index
   def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
-    @comments = @post.comments
+    @comments = @post.comments.includes(:author)
```

Comment:
- [ ] The purpose of eliminating the N+1 query problem, in this case, is to reduce the number of queries needed to fetch, for example, multiple resources associated with another resource. Therefore, using `.includes(:author)` here is somewhat unnecessary as there is only one author. We should this `.includes.(:comments)` instead.